lib/sql-string.js

```diff
@@ -193,14 +193,16 @@ SqlString.bufferToString = function(buffer, dialect) {
 };
 
 SqlString.objectToValues = function(object, timeZone) {
-  var values = [];
-  for (var key in object) {
```

Comment:
In general micro optimizations are only worth it over readability if that piece of code is VERY HOT.
I don't have any numbers on how often objectToValues are called. But generally for inserts and updates i don't believe there's any major hot code path. Compared to reads/selects where we might have 10K rows that need to be processed very performance matters very much.

Have you measured the performance of lodash `forIn` or `forOwn`? lodash implementations are generally fast enough that there's no reason not to use them unless it's a very perf critical piece of code.